src/components/NcSelect/NcSelect.vue

```diff
@@ -1044,6 +1044,7 @@ body {
 
 .vs__dropdown-menu {
 	border-color: var(--color-primary);
+	padding: 5px;
```

Comment:
For the `Actions` component we use a value of `4px`
https://github.com/nextcloud/nextcloud-vue/blob/602c9073bdb3333353744561c897dc3c9d426875/src/components/NcActions/NcActions.vue#L1192

which makes sense, because together with the `border-radius: 6px` used below, it adds up to `var(--border-radius-large) = 10px`.